net/speedtestcpp/Makefile

```diff
@@ -0,0 +1,66 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=speedtestcpp
+PKG_VERSION:=1.20.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/oskarirauta/speedtestcpp/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=3ab42f95eae91614e1e8455bda1d420a60818fbbd9e975244aaaf9caedf96854
+
+PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libspeedtestcpp
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=library for ookla's speedtest
+  DEPENDS:=+libcurl +libstdcpp
+  URL:=https://github.com/oskarirauta/speedtestcpp
+endef
+
+define Package/libspeedtestcpp/description
+  shared library that provides support for ookla's speedtest
+endef
+
+define Package/speedtestcpp
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=SpeedTest++
+  DEPENDS:=+libspeedtestcpp +libstdcpp
+  URL:=https://github.com/oskarirauta/speedtestcpp
+  CONFLICTS:=speedtestpp
+endef
+
+define Package/speedtestcpp/description
+  Yet another unofficial speedtest.net client cli interface
+  forked from taganaka's SpeedTest with few improments and
+  lesser depends.
```

Comment:
Did you try to send it to the original repository? In this case, taganaka's SpeedTest repository?

Reply:
No. Past PR's are not under discussion either. User seems to have vanished. I attempted to contact with e-mail with poor results.